api/client/deals.go

```diff
@@ -27,7 +27,7 @@ type WatchEvent struct {
 
 // Store creates a proposal deal for data using wallet addr to all miners indicated
 // by dealConfigs for duration epochs
-func (d *Deals) Store(ctx context.Context, addr string, data io.Reader, dealConfigs []deals.DealConfig, duration uint64) ([]cid.Cid, []deals.DealConfig, error) {
+func (d *Deals) Store(ctx context.Context, addr string, data io.Reader, dealConfigs []deals.StorageDealConfig, duration uint64) ([]cid.Cid, []deals.StorageDealConfig, error) {
```

Comment:
All changes regarding `StorageDealConfig` are just that, a rename.

Reply:
Been wondering if we should just stop using the word "deal". It could just be "storage" and "retrieval". I have a feeling we started with "deal" just because of words in the grant proposal.

Reply:
But this is fine for now.

Reply:
SGTM